src/controllers/userController.js

```diff
@@ -39,6 +51,17 @@ const getSingleUser = async (req, res) => {
 };
 
 const showCurrentUser = async (req, res) => {
+  /*
+     #swagger.summary = 'Fetch a user by id'
+     #swagger.parameters['id'] = {
+        description: 'Mongo ObjectID of the user to fetch',
+     }
```

Comment:
This endpoint does not take any query or path parameters; it just uses the cookie to find the requesting user's id.